src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/scrollable_message_request.cc

```diff
@@ -65,13 +65,12 @@ ScrollableMessageRequest::~ScrollableMessageRequest() {}
 bool ScrollableMessageRequest::Init() {
   /* Timeout in milliseconds.
      If omitted a standard value of 10000 milliseconds is used.*/
+  uint32_t request_timeout = 0;
   if ((*message_)[strings::msg_params].keyExists(strings::timeout)) {
-    default_timeout_ =
+    request_timeout =
         (*message_)[strings::msg_params][strings::timeout].asUInt();
-  } else {
-    const int32_t def_value = 30000;
-    default_timeout_ = def_value;
   }
+  default_timeout_ += request_timeout;
```

Comment:
@ValeriiMalkov I can see that before there was default value 30000. Is it redundant or we have this default value somewhere else?

Reply:
@ValeriiMalkov @LuxoftAKutsan If `timeout `param omitted a standard value of 30000 milliseconds should be used.

Reply:
@LitvinenkoIra @LuxoftAKutsan Standard value gets from `.ini` file which is 10000. I didn't find in req-t value which is 30000.